src/Angular.js

```diff
@@ -143,6 +144,7 @@ var minErrConfig = {
  * current configuration if used as a getter. The following options are supported:
  *
  * - **objectMaxDepth**: The maximum depth to which objects are traversed when stringified for error messages.
+ * - **urlMaxLength**: The maximum length of the URL reference in the error messages.
```

Comment:
The problem is each parameter value is encoded using `encodeURIComponent`
so when slice this encoded value, the browser (using chrome) found that it is invalid encoding, and replace it with undefined.
```javascript
for (i = 0, paramPrefix = '?'; i < templateArgs.length; i++, paramPrefix = '&') {
      url += paramPrefix + 'p' + i + '=' + encodeURIComponent(templateArgs[i]);

      if (url.length > minErrConfig.urlMaxLength) {
        url = url.substr(0, minErrConfig.urlMaxLength - 3) + '...';
        break;
      }
    }
```

I will try to find a solution that slice the url without harming the encoded data.